webpack.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: path.join(__dirname, 'src', 'index.js'),
+  output: {
+    path: path.join(__dirname, 'dist'),
+    filename: 'index.bundle.js',
+  },
+  mode: process.env.NODE_ENV || 'development',
```

Comment:
you never set NODE_ENV env variable, therefore it'll always be undefined
please use do https://www.npmjs.com/package/dotenv and setup your env variable through package.json CLI scripts